javascript/javascript2/week3/theFastestPresser/theFasterPresser.js

```diff
@@ -0,0 +1,93 @@
+
+ 
+ 
+ const input= document.getElementById('input'); 
+ const button=document.getElementById('game'); 
+ const dCounter=document.getElementById('count-d'); 
+ const kCounter=document.getElementById('count-k'); 
+
+//  document.getElementById('count-d').innerHTML=totald; 
+//  document.getElementById('count-k').innerHTML= totalk; 
+ let total=0;
+ let totald=0; 
+ let totalk=0; 
+ button.addEventListener('click', handleClick); 
+
+ function handleClick( ){
+  const time= input.value; 
+  document.addEventListener('keyup', handlekeypress); 
+
+     setTimeout( endGame, time*1000)
+ }; 
+
+
+function endGame(){
+    document.removeEventListener('keyup', handlekeypress)
+};
+endGame(); 
```

Comment:
You don't need to call endGame() again 'cause you've already done it earlier in setTimeOut. When you're doing this here you execute your function right away before time is up.

Reply:
thanks . i will try to modify that :)